nav/specdev.html

```diff
@@ -106,7 +105,7 @@ <h2>Resources</h2>
 <dt>Language enablement</dt>
 <dd>
 <p>Developers who want to better understand language or writing system features, and what must be taken into account to develop internationalized solutions, can find relevant information using the <a class="highlightLink" href="https://www.w3.org/TR/typography/">Language Enablement Index</a>.</p>
-<p>Other useful resources include the <a class="highlightLink" href="https://www.w3.org/International/i18n-activity/textlayout/">Layout issue tracker</a>, which lists ongoing questions and discussions related to language enablement. Also, the <a class="highlightLink" href="https://w3c.github.io/type-samples/">Type Samples</a> library provides pictures of typographic features in the wild.</p>
+<p>Other useful resources include the <a class="highlightLink" href="https://www.w3.org/International/i18n-activity/textlayout/">Language enablement issue tracker</a>, which lists ongoing questions and discussions related to language enablement. Also, the <a class="highlightLink" href="https://w3c.github.io/type-samples/">Type Samples</a> library provides pictures of typographic features in the wild.</p>
```

Comment:
This paragraph may need rewriting, because we are no longer using the issue tracker page in the same way as we were before.  Now, the links to look for are on the home pages of each of the Language Enablement repositories.